resources/guidelines/testing/store/quality-guidelines-plugins/index.md

```diff
@@ -1,356 +1,373 @@
 
-* Use *Scheme.org's Structured Data Testing Tool* to check the homepage, categories, and various product detail pages
-(incl. available products, unavailable products, products with no review, single review, many reviews with various ratings, out of stock products, products to be released in the future or any other kind of product configuration).
-Also, check for any new bugs.
+###Link to Demoshop
 
-* Do a *Lighthouse Audit* to check the performance and quality of your frontend app.
-There should not be any drastic change in performance or accessibility values when activating the app.
+If you provide a demo shop, the link must be valid (the URL cannot contain http: or https:).
+Do not link to your test environments, as we will delete them automatically two weeks after they are created.
 
-### Frontend apps
+###Personal data protection information
 
-* Links must include a "title-tag" and images must have an "alt-tag".
+If necessary, personal data protection information has to be set.
+If personal data of the customers (store operator and/or his customers) are processed with this extension according to Art. 28 DSGVO, the following information of the data processing company must be stored in the field "Subprocessor".
 
-* If you create custom controller URLs in the sales channel, please note that we check for SEO and a valid canonical-tag.
+If other companies are involved in the data processing of personal data, the same information must be stored accordingly for them in the field "Further subprocessors".
 
-* Use *Scheme.org's Structured Data Testing Tool* to check the homepage, categories, and various product detail pages
-(incl. available products, unavailable products, products with no review, single review, many reviews with various ratings, out of stock products, products to be released in the future or any other kind of product configuration).
-Also, check for any new bugs.
+###Configuration manual
 
-* We check for new errors throughout the entire Storefront using the Browser Debug Console.
-We also pay attention to new JavaScript errors.
+Explain how your app is installed and configured, how it works on a technical base, and how it can be used to achieve the desired result.
+Of course, your app manual should contain a setup guide and be accompanied by clean HTML source code.
 
-* We do a *Lighthouse Audit* to check the performance and quality of your frontend app.
-There should not be any drastic change in performance or accessibility values when activating the app.
+###Manufacturer Profile
 
-### Backend apps
+Your manufacturer profile must mandatorily contain accurate English and German descriptions and a manufacturer logo.
+You can find the manufacturer profile in your account under Shopware Account > Extension Partner > Extension Partner profile.
 
-We check the complete functionality of the app and test wherever the Administration is impacted by the app.
+[extension partner profile](https://account.shopware.com/producer/profile)
```

Comment:
This can be removed!

Reply:
No. you have to leave this link as it guides to a section often forgotten being filled by manufactures. If the profile is not set, the partner will never get an extension at the store. so therefore a deeplink to account is more then necessary.

Reply:
Yes right however, I have already hyperlinked it here : https://github.com/shopware/docs/pull/1508#discussion_r1793347443